[2024-03-11T14:00:00.659+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2021-08-01T00:00:00+00:00 [queued]>
[2024-03-11T14:00:00.753+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2021-08-01T00:00:00+00:00 [queued]>
[2024-03-11T14:00:00.772+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-11T14:00:01.679+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): third_task> on 2021-08-01 00:00:00+00:00
[2024-03-11T14:00:01.838+0000] {standard_task_runner.py:60} INFO - Started process 2773 to run task
[2024-03-11T14:00:02.045+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'third_task', 'scheduled__2021-08-01T00:00:00+00:00', '--job-id', '696', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpeswxkuh7']
[2024-03-11T14:00:02.161+0000] {standard_task_runner.py:88} INFO - Job 696: Subtask third_task
[2024-03-11T14:00:03.092+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v3.third_task scheduled__2021-08-01T00:00:00+00:00 [running]> on host 26e63b1d0d39
[2024-03-11T14:00:04.657+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2021-08-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-08-01T00:00:00+00:00'
[2024-03-11T14:00:04.685+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-11T14:00:04.705+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task3 and I ran after task1 and at the same time as task 2']
[2024-03-11T14:00:04.963+0000] {subprocess.py:86} INFO - Output:
[2024-03-11T14:00:04.999+0000] {subprocess.py:93} INFO - hey, I am task3 and I ran after task1 and at the same time as task 2
[2024-03-11T14:00:05.028+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-11T14:00:05.376+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=third_task, execution_date=20210801T000000, start_date=20240311T140000, end_date=20240311T140005
[2024-03-11T14:00:05.719+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-11T14:00:06.871+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
